docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Best practices: Analyzing shop data"
+description: Learn how to analyze and present shop data.
+last_updated: Oct 24, 2024
+template: back-office-user-guide-template
+related:
+---
+
+Amazon Quicksight is a powerful business intelligence tool. It's highly customizable and has a lot of tools that can't be covered in one document. This document describes a typical use case and how you can start working with analytics, but doesn't cover all the editing options.
+
+For more insights into creating analyses and dashboards, see [Author Workshop](https://catalog.workshops.aws/quicksight/en-US/author-workshop).
```

Comment:
let's remove it please as you give this link in the Conclusion section where it fits better. At the beginning we need to describe our points first and then have this reference for external docs.